src/woo_search/search_index/tests/test_analyzer.py

```diff
@@ -0,0 +1,32 @@
+from woo_search.search_index.tests.base import ElasticSearchTestCase
+
+from ...utils.tests.vcr import VCRMixin
+from ..client import get_client
+from ..index import Document, Publication
+
+
+class AnalyzerTest(VCRMixin, ElasticSearchTestCase):
+    def test_dutch_analyzer(self):
+        text = "Nederlandse tekst om de dutch analyzer te testen."
+        expected_tokens = [
+            "Nederlandse",
+            "tekst",
+            "om",
+            "de",
+            "dutch",
+            "analyzer",
+            "te",
+            "testen",
+        ]
+
+        for document in [Document, Publication]:
```

Comment:
We can also fetch the documents just how we do it during the initalization command, but I think this is more readable and has less moving parts to it.